packages/gatsby-theme-carbon/src/components/HeaderNav/HeaderNav.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { HeaderNavigation } from 'carbon-components-react/lib/components/UIShell';
+import HeaderNavItem from './HeaderNavItem';
+
+import { useNavItems } from '../../util/NavItems';
+
+const HeaderNav = () => {
+  const navItems = useNavItems();
+
+  return (
+    <HeaderNavigation aria-label="Carbon Design System">
+      {navItems.map((item, i) => (
+        <HeaderNavItem items={item.pages} category={item.title} key={i} />
+      ))}
```

Comment:
Generally you only want to use the index as a key [as a last resort](https://reactjs.org/docs/lists-and-keys.html#keys).

```suggestion
      {navItems.map(({pages, title}) => (
        <HeaderNavItem items={pages} category={title} key={title} />
      ))}
```